channeldb/db.go

```diff
@@ -157,6 +157,12 @@ var (
 			number:    16,
 			migration: migration16.MigrateSequenceIndex,
 		},
+		{
+			// Create a top level bucket which holds information
+			// about our peers.
+			number:    17,
+			migration: mig.CreateTLB(peerBucket),
+		},
```

Comment:
Yeah, we've started doing these for top level buckets. Nice because we don't have to always do nil-checks and create them on the fly. The last few migrations have been this type.